build.zig

```diff
@@ -1,17 +1,18 @@
 const std = @import("std");
 
 pub fn build(b: *std.Build) void {
+    const target = b.standardTargetOptions(.{});
     const optimize = b.standardOptimizeOption(.{});
 
     const win = b.dependency("zigwin32", .{});
 
     const injector = b.addExecutable(.{
         .name = "injector",
-        .root_source_file = .{ .path = "src/main.zig" },
-        .target = .{ .os_tag = .windows },
+        .root_source_file = b.path("src/main.zig"),
+        .target = target,
```

Comment:
To fix this issue, you can add this simple line:
`        .target = b.resolveTargetQuery(.{ .os_tag = .windows }),`